src/ui/scss/init/_gui.scss

```diff
@@ -98,6 +98,11 @@ hr {
   }
 }
 
+img,
+a {
```

Comment:
Confident in no consequences to this rule being so universal?

Reply:
97% confidence